index.html

```diff
@@ -51,20 +52,29 @@
       <h2>Introduction</h2>
 
 <p>W3C is a <strong>growing and global community</strong> where participants
-choose to work together, and it is committed to maintaining a positive working environment, where each participant feels appreciated and respected and where everyone adheres to the same high level of standards of personal behavior. In that process we experience differences in language, location, nationality, and experience. In such a diverse environment, 
-misunderstandings and disagreements happen, which in most cases can be resolved
-informally.</p>
+choose to work together. It is committed to maintaining a positive working environment,
+where each participant feels appreciated and respected and where everyone adheres to the same high level of standards of personal behavior. 
+In that process we experience differences in language, location, nationality, and experience. 
+In such a diverse environment, misunderstandings and disagreements happen, which in most cases can be resolved informally.
+Where that does not resolve the problem, this code outlines how W3C will act.</p>
 	    <ul>
 <li> W3C community participants should treat each other with respect and professionalism and be mindful of cultural differences.</li>
 <li>W3C community participants should communicate constructively and avoid insulting, unwelcome or demeaning behavior.</li>
 <li>W3C strictly prohibits discrimination, intimidation, harassment, and bullying of any kind and on any basis.</li>
-<li>W3C will not tolerate abusive behavior in any form, whether it is verbal, physical, sexual, or implied.</li>
+<li>W3C considers abusive behavior in any form unacceptable, whether it is verbal, physical, sexual, or implied.
+        If such behaviour occurs W3C will act to minimise the harm caused to participants and the likelihood of the behaviour being repeated.</li>
```

Comment:
This seems great, but realistically what can W3C do to minimise the harm caused to participants given that in this situation it has already been caused? 

I think what W3C will _actually_ do is:
* provide impartial Ombudspeople who will:
  * assess whether or not the reported incident is a violation of the CEPC;
  * inform all the reported participants in the incident of the result of this assessment;
* keep a confidential record of the assessment, available to [WHO SHOULD HAVE ACCESS?] for [HOW LONG SHOULD WE KEEP IT FOR?];
* potentially take further action in relation to any individuals who have violated the CEPC.

Perhaps we could write this down? Is this list correct and complete?

Reply:
I raised #34 to try to answer the gaps to be filled in here, if this mini-proposal is adopted.